src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -1058,6 +1058,14 @@ namespace winrt::TerminalApp::implementation
         }
     }
 
+    // Method Description:
+    // - See Pane::SnapDimension
+    float TerminalPage::SnapDimension(const bool widthOrHeight, const float dimension) const
+    {
+        const auto focusedTabIndex = _GetFocusedTabIndex();
+        return _tabs[focusedTabIndex]->SnapDimension(widthOrHeight, dimension);
```

Comment:
Rather than having method named `SnapDimension` that sometimes doesn't do its job, it might be cleaner just not to call it at all in `IslandWindow`.

Secondly, that would only disable the 'window' snapping but not 'pane' snapping (so if one splits terminal vertically and changes the window's width, the left pane will still snap, although the whole window would go smoothly). Should it be this way? Maybe make this 3-way setting that would also cover that (like `fullSnap`, `innerSnap`, `noSnap`).